plugins/headlamp-plugin/bin/plugin-management-utils.js

```diff
@@ -0,0 +1,520 @@
+const { table, getBorderCharacters } = require('table');
```

Comment:
I wonder why this separate file is necessary?

Reply:
So that the plugin commands can be added to both headlamp-plugin and plugin-management.